homeassistant/components/sutro/entity.py

```diff
@@ -0,0 +1,11 @@
+"""SutroEntity class."""
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+
+class SutroEntity(CoordinatorEntity):
+    """Representation of a Sutro Entity."""
+
+    def __init__(self, coordinator, config_entry):
+        """Initialize the entity."""
+        super().__init__(coordinator)
+        self.config_entry = config_entry
```

Comment:
The config entry is already available in the coordinator entity. This class is therefore obsolete / not needed.